src/test/resources/features/launchInfo.feature

```diff
@@ -0,0 +1,14 @@
+@api
+Feature: Exact launch info checking
+
+  Background: User is Logged In
+    Given user is logged in
+
+  Scenario Outline: Check exact launch defects info
+    When user makes request to get info of launch with id 3
+    Then user sees that parameter "<infoBlockName>" has value <expectedValue>
+    Examples:
+      | infoBlockName | expectedValue |
+      | productBug    | 4             |
+      | toInvestigate | 10            |
+      | automationBug | 4             |
```

Comment:
Required:
Simple test with precondition (e.g. adding a new widget) using test steps with different input parameter types - list, custom object, data table.
Did you implement it?

Reply:
I thought that user login in the background like a precondition is enough.

Reply:
Do I need to create more tests?

Reply:
Please, add the functionality, described in the Home task:
![image](https://user-images.githubusercontent.com/116157099/201113264-f0fa49bd-bbe8-4698-98da-cbd5fc28b4f2.png)

Reply:
add test with DataTable